pkg/controller/servicemesh/webhookca/controller.go

```diff
@@ -60,6 +60,11 @@ var autoRegistrationMap = map[string]CABundleSource{
 // Add creates a new Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager) error {
+	if common.Config.OLM.WebhookManagementDisabled == true {
+		createLogger().Info("Webhook Config Management is disabled via olm configuration, not adding webhookca to watch list...")
```

Comment:
nit: `not adding webhookca controller to Manager` makes more sense I think

Reply:
Yes. The initial cert management and the eventual renewal is made by external automation in our case.

Reply:
Renamed. Makes sense.